korean/cry/cry-more.md

```diff
@@ -0,0 +1 @@
+울음 소리를 듣고 왔는지 갑자기 나타난 다람쥐! 눈이 마주치자 달아납니다. 어떻게 할까요?
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.